Launcher/config.json

```diff
@@ -194,6 +194,11 @@
   "ftx-api-secret": "",
   "ftx-api-key": "",
 
+  // FTX.US configuration
+  "ftxus-account-tier": "Tier1", // accept "Tier1", "Tier2", "Tier3", "Tier4", "Tier5", "Tier6", "VIP1", "VIP2", "VIP3", "MM1", "MM2", "MM3"
```

Comment:
They have the same tier values? cool 👍

Reply:
Apparently it has more tiers (+7, 8, 9) and only VIP1 and VIP2. Fixed